src/components/App.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
+import { FormattedMessage, injectIntl } from 'react-intl';
 import { Link } from 'react-router-dom';
 
-const App = () => (
+const App = ({ intl }) => (
   <div className="App">
     <header className="App-header">
       <p>
-        Edit <code>src/App.js</code> and save to reload.
+        <FormattedMessage id="home.hello" defaultMessage="Terve" />
       </p>
-      <Link to="/foo">Foo-page</Link>
+      <Link to={`/${intl.locale}/foo`}>Foo-page</Link>
```

Comment:
Can we translate whole urls (that would be nice)? Or extend Link and which enforces our url structure by itself?